app/services/food_service.rb

```diff
@@ -0,0 +1,12 @@
+class FoodService
+  def self.conn
+      Faraday.new("https://api.nal.usda.gov") do |faraday|
+        faraday.params['api_key'] = ENV['foods_api_key']
+      end
+    end
+
+    def self.food_info(food)
+      response = conn.get("/fdc/v1/foods/search?query=#{food}&api_key=#{ENV['foods_api_key']}")
```

Comment:
Since you are including your api key as a param in the `conn` method on line 4, you don't need to include it here, too.